services/cognitiveservices/v1.0/computervision/models.go

```diff
@@ -0,0 +1,556 @@
+// Changes may cause incorrect behavior and will be lost if the code is regenerated.
+
+import (
+	"encoding/json"
+	"github.com/Azure/go-autorest/autorest"
+	"io"
+)
+
+// AzureRegions enumerates the values for azure regions.
+type AzureRegions string
+
+const (
+	// Australiaeast ...
+	Australiaeast AzureRegions = "australiaeast"
+	// Brazilsouth ...
+	Brazilsouth AzureRegions = "brazilsouth"
+	// Eastasia ...
+	Eastasia AzureRegions = "eastasia"
+	// Eastus ...
+	Eastus AzureRegions = "eastus"
+	// Eastus2 ...
+	Eastus2 AzureRegions = "eastus2"
+	// Northeurope ...
+	Northeurope AzureRegions = "northeurope"
+	// Southcentralus ...
+	Southcentralus AzureRegions = "southcentralus"
+	// Southeastasia ...
+	Southeastasia AzureRegions = "southeastasia"
+	// Westcentralus ...
+	Westcentralus AzureRegions = "westcentralus"
+	// Westeurope ...
+	Westeurope AzureRegions = "westeurope"
+	// Westus ...
+	Westus AzureRegions = "westus"
+	// Westus2 ...
+	Westus2 AzureRegions = "westus2"
+)
+
+// Details enumerates the values for details.
+type Details string
+
+const (
+	// Celebrities ...
+	Celebrities Details = "Celebrities"
+	// Landmarks ...
+	Landmarks Details = "Landmarks"
+)
+
+// DomainModels enumerates the values for domain models.
+type DomainModels string
+
+const (
+	// DomainModelsCelebrities ...
+	DomainModelsCelebrities DomainModels = "Celebrities"
+	// DomainModelsLandmarks ...
+	DomainModelsLandmarks DomainModels = "Landmarks"
+)
+
+// ErrorCodes enumerates the values for error codes.
+type ErrorCodes string
+
+const (
+	// BadArgument ...
+	BadArgument ErrorCodes = "BadArgument"
+	// FailedToProcess ...
+	FailedToProcess ErrorCodes = "FailedToProcess"
+	// InternalServerError ...
+	InternalServerError ErrorCodes = "InternalServerError"
+	// InvalidDetails ...
+	InvalidDetails ErrorCodes = "InvalidDetails"
+	// InvalidImageFormat ...
+	InvalidImageFormat ErrorCodes = "InvalidImageFormat"
+	// InvalidImageSize ...
+	InvalidImageSize ErrorCodes = "InvalidImageSize"
+	// InvalidImageURL ...
+	InvalidImageURL ErrorCodes = "InvalidImageUrl"
+	// NotSupportedImage ...
+	NotSupportedImage ErrorCodes = "NotSupportedImage"
+	// NotSupportedLanguage ...
+	NotSupportedLanguage ErrorCodes = "NotSupportedLanguage"
+	// NotSupportedVisualFeature ...
+	NotSupportedVisualFeature ErrorCodes = "NotSupportedVisualFeature"
+	// StorageException ...
+	StorageException ErrorCodes = "StorageException"
+	// Timeout ...
+	Timeout ErrorCodes = "Timeout"
+	// Unspecified ...
+	Unspecified ErrorCodes = "Unspecified"
+)
+
+// Gender enumerates the values for gender.
+type Gender string
+
+const (
+	// Female ...
+	Female Gender = "Female"
+	// Male ...
+	Male Gender = "Male"
+)
+
+// Language1 enumerates the values for language 1.
+type Language1 string
+
+const (
+	// En ...
+	En Language1 = "en"
+	// Zh ...
+	Zh Language1 = "zh"
+)
+
+// OcrLanguages enumerates the values for ocr languages.
+type OcrLanguages string
```

Comment:
Whats is the difference between this and `Language1`?

Reply:
@mcardosos They come from 2 different enums in the Swagger specification for computer vision . `Language` is an enum for the `analyze` (`AnalyzeImage` in Go SDK ).
 
`OcrLanguage` is for the `ocr` (`RecognizerPrintedText` in Go SDK )